### PR TITLE
Added information about Box

### LINKED ISSUE #65: add mention of box as the currently-recommended solution for "intermediate" data
A discussion with ITSD lead to the conclusion that the best current solution to recommend here for the  is to tell internal user to put things on box and use a public link. Medium-term we'll develop a redirect tool that lets it instead be a more descriptive URL, but since the backing store will likely be Box, we can stick with that as the immediate recommendation.  Need to update the "data guide" to reflect this.

cc @stscicrawford 

EDIT: Also see https://jira.stsci.edu/browse/JUSI-26

### PR DESCRIPTION
**Pull request by brechmos-stsci**
Box is an alternative that several groups have used for hosting files that can then be used in automated testing etc.  I added a section in the document to discuss this. 

Fixes #65

### CODE REVIEW

**Comment by brechmos-stsci** [comment]
If this is going to be the "semi-official" way for now, I can add a few screen shots how to get the link if that is desired.

**Comment by eteq** [comment]
@brechmos-stsci - I made some minor changes and also copied over the slightly more detailed instructions from our DMD managed data box folder.  I found it easier to just do the commits locally and push them up, but if you disagree with any of the changes I made feel free to discuss and/or just back them out!

File: guides/where-to-put-your-data.md [hunk 1]
```diff
@@ -32,19 +32,47 @@ of files, so e.g. databases are beyond the scope of this guide.
...
+#### Obtaining Static Box Links
+
+Note that if you are making a static link publicly visible, it is best practice
+to use a shared folder that is managed by a shared data account rather than
+being tied to your user account.  This is also helpful to prevent thiese files
+from counting against your account's size restrictions.  Talk to ITSD or your 
+team to see what, if any shared Box folder you might have.
+
+1. Go to the box web page for a box folder 
+   (https://stsci.app.box.com/folder/######) and upload your file.
```
Comment (pllim) [inline guides/where-to-put-your-data.md]: I don't like the auto-linking of a link that we know does not work
```suggestion
   (`https://stsci.app.box.com/folder/######`) and upload your file.
```

**Comment by brechmos-stsci** [comment]
I am good with any changes you made. feel free to merge when you think it is ready.

**Pull request merged**